test/thorest-api/transactions/post-tx-with-clauses.test.ts

```diff
@@ -0,0 +1,58 @@
+import { Transaction } from '@vechain/sdk-core'
+import { ThorWallet } from '../../../src/wallet'
+import { TransactionDataDrivenFlow } from './setup/transaction-data-driven-flow'
+import {
+    checkTransactionLogSuccess,
+    checkTxInclusionInBlock,
+    compareSentTxWithCreatedTx,
+    successfulPostTx,
+    successfulReceipt,
+} from './setup/asserts'
+
+/**
+ * @group api
+ * @group transactions
+ */
+describe('Send transaction with clauses', function () {
+    const wallet = ThorWallet.withFunds()
+
+    beforeAll(async () => {
```

Comment:
I think this can be removed, `ThorWallet.withFunds()` is actually returning a wallet with funds without the need of waiting for them